sass/_themes.scss

```diff
@@ -1,294 +1,295 @@
 /* stylelint-disable color-no-hex */
 /* stylelint-disable value-keyword-case */
-$COLOR_WHITE       : #ffffff;
-$COLOR_LIGHT_GRAY  : #fafafb;
-$COLOR_GRAY        : #e9e9ed;
-$COLOR_DARK_WHITE  : #cccccc;
-$COLOR_LIGHT_GRAY_2: #b0b3bf;
-$COLOR_BRAND_BLUE_5: #f4f4f6;
-$COLOR_BLUE_GRAY   : #6e7287;
-$COLOR_YELLOW      : #fef1cf;
-$COLOR_LIGHT_ORANGE: #ffc107;
-$COLOR_ORANGE_2    : #ee9e58;
-$COLOR_ORANGE      : #e98024;
-$COLOR_LIGHT_RED   : #f44336;
-$COLOR_GREEN_2     : #66bb6a;
-$COLOR_LIGHT_GREEN : #4caf50;
-$COLOR_GREEN       : #2e8836;
-$COLOR_SKY_BLUE    : #2196f3;
-$COLOR_LIGHT_BLUE  : #1565c0;
-$COLOR_BLUE        : #2a3052;
-$COLOR_DARK_BLUE   : #15212d;
-$COLOR_DARK_GRAY   : #d2d3da;
-$COLOR_LIGHT_BLACK : #5c5c5c;
-$COLOR_BLACK       : #000000;
-$COLOR_ORANGE      : #ff9933;
-
-// Dark Theme Colors
-$COLOR_BRANDBLUE_5 : #f4f4f6;
-$COLOR_BRANDBLUE_40: #aaacba;
-$COLOR_BRANDBLUE_75: #60647e;
-$COLOR_BRANDBLUE_85: #4a4f6c;
-$COLOR_BRANDBLUE_95: #353b5b;
+$COLOR_WHITE        : #ffffff;
+$COLOR_BLACK        : #000000;
+$COLOR_LIGHT_BLACK_1: rgba($COLOR_BLACK, 0.8);
+$COLOR_LIGHT_BLACK_2: rgba($COLOR_BLACK, 0.4);
```

Comment:
@ashkanx i coulnt find where do we use this `COLOR_LIGHT_BLUE_1` to `COLOR_LIGHT_BLUE_5`

Reply:
@MahboobehMohammadi I added the all colours according to Zeplin projects for both light and dark themes. so these are used later